llvm/test/CodeGen/WebAssembly/ref-type-mem2local.ll

```diff
@@ -1,4 +1,5 @@
 ; RUN: opt < %s -wasm-ref-type-mem2local -S | FileCheck %s
```

Comment:
Since we always want llc to run this pass and can test it this way, do we still need the standalone opt test too?

Reply:
Removed.